go.mod

```diff
@@ -47,8 +45,8 @@ require (
 	github.com/fsnotify/fsnotify v1.5.4 // indirect
 	github.com/go-logr/logr v1.2.3 // indirect
 	github.com/go-openapi/jsonpointer v0.19.5 // indirect
-	github.com/go-openapi/jsonreference v0.19.5 // indirect
-	github.com/go-openapi/swag v0.19.14 // indirect
+	github.com/go-openapi/jsonreference v0.20.0 // indirect
```

Comment:
If none of the direct dependencies are changing, how are the indirect ones changing? Should this and the sum file really be committed?

Reply:
Agreed as well.  I wasn't changing any of the dependencies so I was confused when these files changed.  I will make sure they match the `main` branch.

Reply:
Fixed.  Not sure why they were different.